smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/integration/HttpBindingProtocolGenerator.java

```diff
@@ -374,13 +374,13 @@ private void writeHeaders(
 
         if (!documentBindings.isEmpty()) {
             // Write the default `body` property.
-            writer.write("let body: any = {};");
+            writer.write("let body: any;");
```

Comment:
This needs an equivalent fix needed in the `HttpRpcProtocolGenerator`.

Reply:
Added in 0bf9184.

I verified that `HttpRequest` will properly handle this as part of testing with aws/aws-sdk-js-v3/pull/844 to confirm fix for aws/aws-sdk-js-v3/issues/838